docs/inbound-agent.md

```diff
@@ -21,7 +21,7 @@ The long string of hex digits is a secret key that the client needs to establish
 Depending on how you connect you may also need the agent name, "AgentName" in this example, or the (misnamed) jnlpUrl, "https://myjenkins.example.com/computer/AgentName/jenkins-agent.jnlp".
 
 ### "Launch" button
-This is the only launch mechanism that actually uses JNLP.
+This is the only launch mechanism that actually uses Java Web Start and the JNLP files.
```

Comment:
The last part is incorrect. The item below, "Download JNLP file", clearly uses the JNLP file, though it doesn't use JNLP (or Web Start).